linera-views/src/rocks_db.rs

```diff
@@ -453,38 +453,56 @@ pub fn create_rocks_db_common_config() -> CommonStoreConfig {
     }
 }
 
+/// A path and the guard for the temporary directory if needed
+#[derive(Clone, Debug)]
+pub struct PathDir {
```

Comment:
I find `PathDir` a bit confusing; maybe `RocksDbPath` or `DirWithGuard` or something like that?

Reply:
`PathDir` is simply not correct in English. What was intended is probably `DirPath` or `DirectoryPath`.

Reply:
I think `PathWithGuard` should be fine then.